src/main/java/movie/tickets/security/AuthenticationServiceImpl.java

```diff
@@ -25,7 +25,7 @@ public AuthenticationServiceImpl(UserService userService,
     @Override
     public User login(String email, String password) throws AuthenticationException {
         logger.info("User with email " + email + " is trying to login");
-        Optional<User> userFromDb = userService.findByEmail(email);
+        Optional<User> userFromDb = Optional.ofNullable(userService.findByEmail(email));
```

Comment:
```suggestion
        Optional<User> userFromDb = Optional.ofNullable(userService.findByEmail(email));
```
userService.findByEmail(email) will ever return null?

Reply:
nope